### DISCUSSION TITLE
Connecting to existing database [May 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2243] [Discord Threads]
> 🙋‍♂️ **Tastier Sub**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### NocoDB stores its meta information always in NC_DB. You can find details here https://docs.nocodb.com/engineering/architecture/   
####    
#### Newer tables created in the project (after connecting to external DB) will be part of external DB  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978830476338921523)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
